### PR TITLE
[Build] Add Travis CI Integration

### PR DESCRIPTION
**Pull request by emmabukacek**
## Issues
* #3 

## Changes
* Add .travis.yml
* Add https://github.com/lpil/dogma for linting

@sethetter @kaylah @blad

getinsertpic.com

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  - mix local.hex --force
+
+install:
+  - cd api/
+  - mix deps.get
+  - mix do ecto.create, ecto.migrate
+
+script:
+  - mix test
+  - mix dogma
```
Comment (sethetter) [inline .travis.yml]: You should switch the order of these, dogma will probably always run faster and it'd suck to have a long test process succeed just to fail on a style issue.

Comment (emmabukacek) [inline .travis.yml]: Ooo! Good call; I'll make the change for sure.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+language: elixir
+sudo: false
```
Comment (blad) [inline .travis.yml]: Do we need to define the sudo setting? It seems like it should already be `false` as a default. https://docs.travis-ci.com/user/ci-environment/

Comment (emmabukacek) [inline .travis.yml]: I'll try without it!

**Pull request merged**